crates/ruma-client-api/src/message/send_message_event.rs

```diff
@@ -36,8 +36,8 @@ pub mod v3 {
 
             /// The transaction ID for this event.
             ///
-            /// Clients should generate an ID unique across requests with the
-            /// same access token; it will be used by the server to ensure
+            /// Clients should generate an ID unique across requests within the
+            /// same session; it will be used by the server to ensure
```

Comment:
I feel like we should clarify what "session" means here, then.

Reply:
What do you think of the docs now?